numpy/core/include/numpy/ndarraytypes.h

```diff
@@ -1870,9 +1889,12 @@ typedef void (PyDataMem_EventHookFunc)(void *inp, void *outp, size_t size,
          * NOTE: We could make a copy to detect changes to `f`.
          */
         PyArray_ArrFuncs *f;
-    } PyArray_DTypeMeta;
 
-    #define NPY_DTYPE(descr) ((PyArray_DTypeMeta *)Py_TYPE(descr))
+        /* DType methods, these could be moved into its own struct */
```

Comment:
Could this get an "added
```suggestion
        / *
          * DType methods, these could be moved into its own struct. Added
          * in version 1.20
         */
```